Additions to CHANGELOG (#70)
swift-server/vscode-swift: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

## 0.1.0

### Added

- Automatically create build tasks based on the targets in a package.
- Show package dependencies in the Explorer.
- Add button to package dependencies view to update packages.
- Resolve dependencies when **Package.swift** or **Package.resolved** changes.

> - Integrated with Soucekit-LSP.

Reviewer: ```suggestion
- Integrate the SourceKit-LSP extension.
```

Author: That implies it is a separate extension.

> - Generate launch configurations for each executable and tests in Package.swift.

Reviewer: ```suggestion
- Generate launch configurations for each executable and tests in **Package.swift**.
```

> - Added "Swift" output channel providing a history of all actions.

Reviewer: ```suggestion
- Add "Swift" output channel providing a history of all actions.
```

Author: Thinking about it. Shouldn't all the lines be in the past tense. Given the section is called "Added".

> - Add status bar item for when swift resolve or updates are running.

Reviewer: ```suggestion
- Add status bar item for when resolve or update tasks are running.
```

- Bundle using ESBuild.

> - Add configuration parameter `path` to define where swift is found.

Reviewer: ```suggestion
- Add configuration parameter `path` to define where `swift` is found.
```

Author: Not sure we need to add marks around swift but it should be capitalised.
